ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,25 @@
+passenger = (12, True, "Bonnell, Miss. Elizabeth", "female", 58)
+print('passenger: ', passenger)
+name = passenger[2]
+age = passenger[-1]
+print('Name: ', name, 'Age: ', age)
+survived_and_name = passenger[1:3]
+gender_and_age = passenger[3:5]
```

Comment:
While getting `gender_and_age`, only specify the start of the index range:

---

![image](https://user-images.githubusercontent.com/16143771/148659834-cadc558a-4c4d-4371-9c4c-8324af11ecce.png)